AWapi.playground/Contents.swift

```diff
@@ -0,0 +1,101 @@
+//: Playground - noun: a place where people can play
+
+import UIKit
+import PlaygroundSupport
+
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+var str = "Hello, playground"
```

Comment:
What is this playground stuff?

Reply:
In Python people often keep notebooks in a repo until all the core functionality is figured out. Also I forgot to add .playground to gitignore